### DISCUSSION TITLE
help. I was trying to run it on Windows

### DISCUSSION THREAD

**Discussion by Naiml007** [Ani-Moopa/Moopa#110] [General]
![image](https://github.com/Ani-Moopa/Moopa/assets/90306726/c5ea1ffd-2785-4464-8e9a-6c87f3de67f6)

![image](https://github.com/Ani-Moopa/Moopa/assets/90306726/3dddec04-b8e3-4fb5-8931-2cf32130920d)

**Comment by Naiml007** [comment]
@DevanAbinaya

**Comment by Naiml007** [comment]
![Screenshot 2024-01-08 223610](https://github.com/Ani-Moopa/Moopa/assets/90306726/26d42e42-26d8-4554-b817-d0fabb7393f5)

**Comment by DevanAbinaya** [comment]
Something's wrong with your API, please check again and make sure it works

**Reply by Naiml007** [reply]
![image](https://github.com/Ani-Moopa/Moopa/assets/90306726/49b4d96e-d865-4da4-9dd4-1c0fa1130c82)

![image](https://github.com/Ani-Moopa/Moopa/assets/90306726/cc203c84-8795-4034-b8a4-179d20ba10cf)

**Comment by Naiml007** [comment]
proxy issue ?

![image](https://github.com/Ani-Moopa/Moopa/assets/90306726/cad8a50c-cff0-431b-9c1f-18b0677074da)

**Comment by Naiml007** [comment]
m3u8 proxy error

![image](https://github.com/Ani-Moopa/Moopa/assets/90306726/16e15455-c0a9-4f6e-927b-4c7ebf852ad2)
